monkey/monkey_island/cc/resources/pba_file_upload.py

```diff
@@ -74,6 +80,9 @@ def delete(self, file_type):
         :param file_type: Type indicates which file was deleted, linux of windows
         :return: Empty response
         """
+        if self.check_file_type(file_type):
+            return Response(status=422, mimetype="text/plain")
```

Comment:
Can't we get the status code expressed as a meaningful const? [Python has this build in](https://docs.python.org/3/library/http.html#http-status-codes) and UNPROCESSABLE_ENTITY is more readable than 422.